packages/core/helper-plugin/lib/src/components/GenericInput/storyUtils/layout.js

```diff
@@ -135,7 +135,7 @@ const layout = [
           key: 'pizza',
           value: 'pizza',
           metadatas: {
-            intlLabel: { id: 'pizza-label', defaultMessage: 'Pizzza' },
+            intlLabel: { id: 'pizza-label', defaultMessage: 'Pizza' },
```

Comment:
I didn't see any particular reason for the extra 💤 .

Reply:
Because we like pizzzzzzaaaas

Reply:
Fwiw, if you had more 💤, I'd have left it alone 😉 . -- Seriously, if things are clearly intentional/cute as opposed to very close to a misspelling, I leave them alone. -- I like 🍕 too.